addons/mail/tests/discuss/test_discuss_channel.py

```diff
@@ -378,8 +389,20 @@ def test_channel_write_should_send_notification_if_image_128_changed(self):
                 "type": "mail.record/insert",
                 "payload": {
                     'Thread': {
-                        "avatarCacheKey": avatar_cache_key,
-                        "id": channel.id,
+                        'avatarCacheKey': avatar_cache_key,
+                        'channel_type': channel.channel_type,
+                        'memberCount': channel.member_count,
+                        'id': channel.id,
+                        'name': channel.name,
+                        'defaultDisplayMode': channel.default_display_mode,
+                        'description': channel.description,
+                        'uuid': channel.uuid,
+                        'state': 'open',
+                        'is_editable': channel.is_editable,
+                        'group_based_subscription': bool(channel.group_ids),
+                        'create_uid': channel.create_uid.id,
+                        'authorizedGroupFullName': channel.group_public_id.full_name,
+                        'allow_public_upload': channel.allow_public_upload,
```

Comment:
as we are already comparing keys before/after, we should probably just send the keys that changed (and model + id)